pkgs/applications/networking/flottbot/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib, buildGoModule, fetchFromGitHub, nixosTests, nix-update-script }:
+buildGoModule rec {
+  pname = "flottbot";
+  version = "0.12.0";
+
+  src = fetchFromGitHub {
+    owner = "target";
+    repo = pname;
+    rev = version;
+    sha256 = "538c2SkyOYmm84yM0svbPeJtSIMHRI6mot8MIP5wBfc=";
+  };
+
+  vendorSha256 = "sha256-6j1nQBlFqd78v/CNV7ivwYZ5CEzGh+An6xcQr4dAkAs=";
+
+  subPackages = [ ];
```

Comment:
What do you need this for?

Reply:
Probably copy/paste from another Go package, sorry. 😅

Reply:
Could you delete it?